www/pages/home/Home.css

```diff
@@ -0,0 +1,7 @@
+@import "../../shared/styles/variables";
```

Comment:
Check this comment: https://github.com/moxystudio/next-with-moxy/pull/3#discussion_r329499038

Reply:
I'm not sure if there's been more discussion on this, but i personally find it awkward to import 'imports', which is very nondescript, and would consider a different name. Also, in general, though i agree with not having to hunt for files in imports, i would personally lean towards specificity than generalization. What do you think?